README.md

```diff
@@ -20,9 +20,9 @@ B --> C[Remove]
 
 ## Contributing
 
-This project welcomes contributions and suggestions.  Most contributions require you to agree to a
+This project welcomes contributions and suggestions. Most contributions require you to agree to a
 Contributor License Agreement (CLA) declaring that you have the right to, and actually do, grant us
-the rights to use your contribution. For details, visit https://cla.opensource.microsoft.com.
+the rights to use your contribution. For details, visit https://github.com/Azure/benchpress.
```

Comment:
Please keep the original link that links to https://cla.opensource.microsoft.com

Reply:
markdown-link-check is seeing some of the links as deadlink and giving false error. I will find a way to workaround